src/core/public/doc_links/doc_links_service.ts

```diff
@@ -46,10 +46,84 @@ export class DocLinksService {
     const OPENSEARCH_DOCS = `https://opensearch.org/docs/opensearch/`;
     const OPENSEARCH_DASHBOARDS_DOCS = `https://opensearch.org/docs/dashboards/`;
 
+    const OPENSEARCH_VERSIONED_DOCS =
+      OPENSEARCH_WEBSITE_URL + `docs/` + DOC_LINK_VERSION + `/opensearch/`;
+    const OPENSEARCH_DASHBOARDS_VERSIONED_DOCS =
+      OPENSEARCH_WEBSITE_URL + `docs/` + DOC_LINK_VERSION + `/dashboards/`;
+
```

Comment:
Can we use template strings here? e.g. `${OPENSEARCH_WEBSITE_URL}docs/${DOC_LINK_VERSION}/opensearch/`

Reply:
will o